lib/strings.nix

```
@@ -144,6 +144,19 @@ rec {
   */
   concatLines = concatMapStrings (s: s + "\n");
 
+  /* Replicate a string n times, and concatenate the parts into
+     a new string.
+
+  Type: replicateString :: int -> string -> string
```

```suggestion
  /*
    Replicate a string n times,
    and concatenate the parts into a new string.

    Type: replicateString :: int -> string -> string
```

Consistent indentation and [semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).